analysis/scripts/2233-power-exploration.Rmd

```diff
@@ -0,0 +1,563 @@
+---
+title: "An Exploration of Power calculations"
```

Comment:
@bpffjl  This file is just a demo. I suspect we should put its contents in the SOP or some place as an example of two ways to do power analysis. At least I think it works. Would be happy to discuss it.